src/routes/safe/components/Apps/utils.ts

```diff
@@ -58,16 +56,19 @@ export const getAppInfoFromUrl = memoize(async (appUrl: string): Promise<SafeApp
     loadingStatus: SAFE_APP_FETCH_STATUS.ERROR,
   }
 
-  if (!appUrl?.length) {
+  if (!appUrl?.length || !isValidURL(appUrl.trim())) {
     return res
   }
 
-  res.url = appUrl.trim()
-  const noTrailingSlashUrl = removeLastTrailingSlash(res.url)
+  res.url = new URL(appUrl.trim()).origin
 
   let appInfo: AppData | undefined
   try {
-    appInfo = await axios.get(`${noTrailingSlashUrl}/manifest.json`, { timeout: 5_000 })
+    appInfo = await axios.post(
```

Comment:
if this is getting shouldnt it be a `get` ?

Reply:
I felt like passing urls as a get params was just a recipe for trouble. Plus it's actually a mutating operation (we're storing the manifest in a cache of the client-gateway), so post makes sense too IMHO